py2store/persisters/sqlserver_w_pyodbc.py

```diff
@@ -0,0 +1,121 @@
+import platform
+import subprocess
+from collections.abc import MutableMapping
+
+
+class SQLServerPersister(MutableMapping):
+    def __init__(self, conn_protocol='tcp', host='localhost', port='1433', db_username='SA', db_pass='Admin123x',
+                 db_name='py2store', table_name='person', primary_key='id', data_fields=('name',)):
```

Comment:
`primary_key` might be a tuple of multiple key fields. Persister should be flexible enough to work with such cases.

Also, please rename it to `key_fields`.